docs/BreakingChanges/108 GroupItem properties incorrectly changed while leaving the visual tree.md

```diff
@@ -0,0 +1,32 @@
+## 108: GroupItem properties incorrectly changed while leaving the visual tree
```

Comment:
{
  "Id": "108",
  "Title": "GroupItem properties incorrectly changed while leaving the visual tree",
  "Details": "",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "TBD: Following up with feature team"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/658365",
  "Notes": "This is a tricky one because the affects can be far removed from the bug. I'm following up with the feature team to understand what sort of detection/prevention can be done here, if any.\nSource analyzer status: Not planned? Maybe include this in an uber-WPF analyzer that just looks for WPF usage and lists issues like this?",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] TBD: Following up with feature team